tests/framework/framework.go

```diff
@@ -225,15 +237,15 @@ func (f *Framework) GetCdiClient() (*cdiClientset.Clientset, error) {
 }
 
 func (f *Framework) GetKubeClient() (*kubernetes.Clientset, error) {
-	return GetKubeClientFromFlags(f.Master, f.KubeConfig)
+	return f.GetKubeClientFromFlags(f.Master, f.KubeConfig)
```

Comment:
if you're changing this to a method then you do not need to pass the 2 args here since they are defined in the receiver.